src/components/ProgressionChecker/Dashboard.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Card, Progress, Typography } from "antd";
+import { useSpring, animated } from "react-spring";
+import DegreeComponentCard from "./DegreeComponentCard";
+import LiquidProgressChart from "./LiquidProgressChart";
+import RadialBarChart from "./RadialBarChart";
+
+function Dashboard() {
+  const { Title, Text } = Typography;
+  const props = useSpring({
+    to: { opacity: 1 },
+    from: { opacity: 0 },
+    delay: 200,
+  });
+  return (
+    <div
+      style={{
+        height: "100vh",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <animated.div className="centerCol" style={props}>
+        {/* <RadialBarChart completedUOC={108} totalUOC={216} /> */}
+        <LiquidProgressChart completedUOC={108} totalUOC={216} />
+        <a
+          href="https://www.handbook.unsw.edu.au/undergraduate/programs/2021/3767?year=2021"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="textLink"
+        >
+          <Title
+            className="text textLink"
+            style={{ marginBottom: "0.5em", marginTop: "0.8em" }}
```

Comment:
Don't put style here

Reply:
With my latest update, I was able to remove all the in-line styling in the code except for the skeleton components which for some reason cant be changed via css classname and requires the `style: {{ }}`.